### ISSUE TITLE
some demo controls not working

### ISSUE THREAD

**Issue by dciccale** [VictorBjelkholm/ngProgress#73]
`increment`, `change height` and `change color` are throwing errors in the console and not working.

when clicking on `increment`

```js
TypeError: ngProgress.status is not a function
    at Object.$scope.count (http://victorbjelkholm.github.io/ngProgress/:369:47)
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:6129:19
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:12496:13
    at Object.Scope.$eval (https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:7770:28)
    at Object.Scope.$apply (https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:7850:23)
    at HTMLAnchorElement.<anonymous> (https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:12495:17)
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:1930:10
    at Array.forEach (native)
    at forEach (https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:110:11)
    at HTMLAnchorElement.eventHandler (https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:1929:5)
```

```js
$scope.count = function($event) {
  $event.preventDefault();
  $scope.progressbar.set(ngProgress.status() + 9); // .status() is not a function
}
```

---

when clicking on `change height`

```js
TypeError: Cannot read property 'preventDefault' of undefined
    at Object.$scope.new_height (http://victorbjelkholm.github.io/ngProgress/:378:19)
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:6129:19
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:12496:13
    at Object.Scope.$eval (https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:7770:28)
    at Object.Scope.$apply (https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:7850:23)
    at HTMLAnchorElement.<anonymous> (https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:12495:17)
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:1930:10
    at Array.forEach (native)
    at forEach (https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:110:11)
    at HTMLAnchorElement.eventHandler (https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:1929:5)
```

```js
$scope.new_height = function(new_height, $event) {
  $event.preventDefault(); // $event is not defined
  $scope.progressbar.height(new_height);
}
```

---

when clicking on `change color`

```js
TypeError: Cannot read property 'preventDefault' of undefined
    at Object.$scope.new_color (http://victorbjelkholm.github.io/ngProgress/:373:19)
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:6129:19
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:12496:13
    at Object.Scope.$eval (https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:7770:28)
    at Object.Scope.$apply (https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:7850:23)
    at HTMLAnchorElement.<anonymous> (https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:12495:17)
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:1930:10
    at Array.forEach (native)
    at forEach (https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:110:11)
    at HTMLAnchorElement.eventHandler (https://ajax.googleapis.com/ajax/libs/angularjs/1.0.1/angular.js:1929:5)
```

```js
$scope.new_color = function(color, $event) {
  $event.preventDefault(); // $event is not defined
  $scope.progressbar.color(color);
}
```

---

i could take a deeper look later

**Comment by VictorBjelkholm** [comment]
Hm, seems like some of the interface of the library have changed while the website doesn't reflect this. Thanks for checking it out, I'll take a look later today at it.

Again, thanks for reporting it!

**Comment by VictorBjelkholm** [comment]
Fixed in 99055a328fffea032d6b18d8ce5664f338594cfe, cd178d5927437f2d88ac65579f8c32d38a051928 and 1ebcd66727ba1e78e7adfedb285f8c4f4314668c

Thanks again for reporting!
